static/styles/settings.css

```diff
@@ -279,6 +279,7 @@ td .button {
     .table-striped {
         table-layout: auto;
         border-collapse: separate;
+        width: 20%;
```

Comment:
This CSS change affects the entire settings UI, not just this panel.  It almost certainly breaks things.

Reply:
thanks, I've solved it now, can you please can a look at it.